rfc3164/machine.go.rl

```diff
@@ -161,7 +178,7 @@ fail := (any - [\n\r])* @err{ fgoto main; };
 
 # note > some BSD syslog implementations insert extra spaces between "PRI", "Timestamp", and "Hostname": although these strictly violate RFC3164, it is useful to be able to parse them
 # note > OpenBSD like many other hardware sends syslog messages without hostname
-main := pri sp* (timestamp | (rfc3339 when { m.rfc3339 })) sp+ (hostname sp+)? msg '\n'?;
+main := pri sp* sequence? (timestamp | (rfc3339 when { m.rfc3339 })) ':'? sp+ (hostname sp+)? msg '\n'?;
```

Comment:
I think that `':'?` should be evaluated only when m.sequence is true, too.

Like you did for the `sequence` rule.

Reply:
Made that change here https://github.com/leodido/go-syslog/pull/22/commits/f5d2a5e31eed61be397c0e2876afeef2b239a7ca